example/ios/Podfile

```diff
@@ -32,15 +22,11 @@ target 'KlaviyoReactNativeSdkExample' do
 
   use_react_native!(
     :path => config[:reactNativePath],
-    # Enables Flipper.
-    #
-    # Note that if you have use_frameworks! enabled, Flipper will not work and
-    # you should disable the next line.
-#    :flipper_configuration => flipper_config,
-    # An absolute path to your application root.
     :app_path => "#{Pod::Config.instance.installation_root}/.."
   )
 
+  pod 'KlaviyoSwift', :path => '../../../klaviyo-swift-sdk/'
```

Comment:
pointing to local version since the changes we made on iOS are not yet available via a release tag

Reply:
maybe add a comment in the code explaining the reason for this line, and instructing to remove it after we release SDK v4.0.0?

Reply:
Good callout. Done.